easybuild/easyblocks/i/icc.py

```diff
@@ -165,59 +142,39 @@ def make_module_req_guess(self):
-            guesses['LD_LIBRARY_PATH'].append('lib/ia32')
-
-        else:
-            # 64-bit toolkit
-            guesses['PATH'].extend([
-                'bin/intel64',
-                'debugger/gdb/intel64/bin',
-                'ipp/bin/intel64',
-                'mpi/intel64/bin',
-                'tbb/bin/emt64',
-                'tbb/bin/intel64',
-            ])
-
-            # in the end we set 'LIBRARY_PATH' equal to 'LD_LIBRARY_PATH'
-            guesses['LD_LIBRARY_PATH'].extend([
-                'compiler/lib/intel64',
-                'debugger/ipt/intel64/lib',
-                'ipp/lib/intel64',
-                'mkl/lib/intel64',
-                'mpi/intel64',
-                'tbb/lib/intel64/%s' % get_tbb_gccprefix(os.path.join(self.installdir, 'tbb/lib/intel64')),
-            ])
-
-            if LooseVersion(self.version) < LooseVersion('2016'):
-                prefix = 'composer_xe_%s' % self.version
-                # for some older versions, name of subdirectory is slightly different
-                if not os.path.isdir(os.path.join(self.installdir, prefix)):
-                    cand_prefix = 'composerxe-%s' % self.version
-                    if os.path.isdir(os.path.join(self.installdir, cand_prefix)):
-                        prefix = cand_prefix
-
-                # debugger is dependent on $INTEL_PYTHONHOME since version 2015 and newer
-                if LooseVersion(self.version) >= LooseVersion('2015'):
-                    self.debuggerpath = os.path.join(prefix, 'debugger')
-
-            else:
-                # new directory layout for Intel Parallel Studio XE 2016
-                # https://software.intel.com/en-us/articles/new-directory-layout-for-intel-parallel-studio-xe-2016
-                prefix = self.comp_libs_subdir
-                # Debugger requires INTEL_PYTHONHOME, which only allows for a single value
-                self.debuggerpath = 'debugger_%s' % self.version.split('.')[0]
-
-                guesses['LD_LIBRARY_PATH'].extend([
-                    os.path.join(self.debuggerpath, 'libipt/intel64/lib'),
-                    'daal/lib/intel64_lin',
-                ])
-
-            # 'lib/intel64' is deliberately listed last, so it gets precedence over subdirs
-            guesses['LD_LIBRARY_PATH'].append('lib/intel64')
+        # 64-bit toolkit
+        guesses['PATH'].extend([
+            'bin/intel64',
+            'debugger/gdb/intel64/bin',
+            'ipp/bin/intel64',
+            'mpi/intel64/bin',
+            'tbb/bin/emt64',
+            'tbb/bin/intel64',
+        ])
+
+        # in the end we set 'LIBRARY_PATH' equal to 'LD_LIBRARY_PATH'
+        guesses['LD_LIBRARY_PATH'].extend([
+            'compiler/lib/intel64',
+            'debugger/ipt/intel64/lib',
+            'ipp/lib/intel64',
+            'mkl/lib/intel64',
+            'mpi/intel64',
+            'tbb/lib/intel64/%s' % get_tbb_gccprefix(os.path.join(self.installdir, 'tbb/lib/intel64')),
```

Comment:
```suggestion
            'tbb/lib/intel64/gcc4.8',
```

`get_tbb_gccprefix` returns `gcc4.8` for the `icc` (or `iccifort`) used in `2020a` and `2020b`. For `2019b`, and older, it'll return `gcc4.4`. Due to the age of them I propose just hard-coding here and that will remove the import.

Reply:
two quick thoughts;
1. I would like to start testing and merging this, as this PR is needed for other things. I think we can clean this up in a later PR if necessary? For this reason, i'm suggesting we skip doing this change in this PR.
2. What is tbb changes prefixes in the future? Surely it's not always going to be gcc 4.8 (I would hope)

Reply:
1. Yes, agreed - we'll do additional cleanup separate to this.
2. I am suggesting the hard-coding in the `icc` easyblock but we may as well just leave it as it is and archive it in the future when we've archived off all the `icc` / `iccifort` easyconfigs.